src/Nowruz/modules/general/components/SearchDropdown/index.tsx

```diff
@@ -77,7 +77,7 @@ export const SearchDropdown: React.FC<SelectProps> = ({
   return (
     <div className={`${css.container} ${className}`}>
       <div className={css.labelContainer}>
-        <label htmlFor={id} className={css.label} onClick={handleLabelClick} aria-describedby={id}>
+        <label htmlFor={id} className={css.label} onClick={handleLabelClick} aria-describedby={id} id="searchDropdown">

```

Comment:
I suggest id=`searchDropDown-${id}`, because we might have several of these component in a single form